db/local/db.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+import os
+import subprocess
+from typing import Type
+
+from peewee_migrate import Router
+
+from src.base_model import BaseModel, db
+
+MIGRATION_DIRECTORY: str = "db/local/migrations"
+
+models: list[Type[BaseModel]] = BaseModel.__subclasses__()
+
+
+def run(args: argparse.Namespace):
+    """Run the database
+
+    :param args: parsed arguments containing the string `database_name`
+    :type args: argparse.Namespace
+    """
+    name = args.database_name
+    print("Running database...")
+    print(os.getcwd())
+    subprocess.run(
+        [
+            "docker-compose",
+            "-f",
+            "db/local/docker-compose.yml",
+            "up",
+            "-d",
+            "-".join(["postgres", name]),
+        ],
+        check=True,
+    )
+
+
+def stop(args: argparse.Namespace):
+    """Stop the database
+
+    :param args: parsed arguments containing the string `database_name`
+    :type args: argparse.Namespace
+    """
+    name = args.database_name
+    print("Stopping database...")
+    subprocess.run(
+        [
+            "docker-compose",
```

Comment:
Depending on your docker install, this could also be `docker compose`. `docker-compose` is the older version (See: <https://docs.docker.com/compose/compose-v2/#differences-between-compose-v1-and-compose-v2>)

Reply:
Good catch! Should we use v2?

Reply:
Well v1 will be deprecated end  of june 2023, so yes.

Reply:
👍